tools/deb/description

```diff
@@ -0,0 +1 @@
+Istio proxy provides the envoy sidecar used by the http://istio.io project.
```

Comment:
(same nit again) vi this file (add a newline)

ps: please go over all my previous comments - tia

Reply:
ping :-)

Reply:
Done. Just curious, why is CRLF at end required ?

Reply:
only LF - because that's how most source code file are afaik, they are line based and contains N lines not N-1 ?